Sources/Models/CalculationParameters.swift

```diff
@@ -79,3 +82,253 @@ public struct CalculationParameters: Codable, Equatable {
         }
     }
 }
+
+public extension CalculationParameters {
+    /// Returns the recommended calculation parameters for the specified country code.
+    static func recommended(forCountryCode code: String) -> CalculationParameters? {
+        mapped.first { $0.value.contains(code) }?.key.params
+    }
+
+    #if canImport(CoreLocation)
+    /// Returns the recommended calculation parameters  for the specified geographic data.
+    static func recommended(for placemark: CLPlacemark) -> CalculationParameters? {
+        guard let countryCode = placemark.isoCountryCode else { return nil }
+        return recommended(forCountryCode: countryCode)
+    }
+    #endif
+
+    private static let mapped: [CalculationMethod: [String]] = [
+        .egyptian: ["EG", "LY", "SD"],
```

Comment:
for maintainability lets do this actually:

```
.egyptian: [
    "EG", // Egypt
    "LY", // Libya 
    "SD" // Sudan
],
```

Reply:
Moved to JSON which cannot have comments I believe, but since these are ISO country code may not be necessary.